zerver/openapi/zulip.yaml

```diff
@@ -19598,6 +19711,47 @@ components:
           description: |
             A boolean specifying whether the [invitation](/help/invite-new-users) is a
             reusable invitation link or an email invitation.
+    InvitationInviteExpiresInMinutes:
```

Comment:
You don't need both `Invitation` and `Invite` here. I think this can be simplified to `InviteExpirationParameter`.